src/useNextSanityImage.ts

```diff
@@ -123,6 +142,7 @@ export function useNextSanityImage(
 		}
 
 		const originalImageDimensions = getImageDimensions(id);
+		const croppedImageDimensions = getCroppedDimensions(image);
 
 		const loader: ImageLoader = ({ width, quality }) => {
 			return (
```

Comment:
Perhaps we should also pass the croppedImageDimensions to the imageBuilder callback, so it can be used to modify the image further.

Reply:
Make sure to also modify the `UseNextSanityImageBuilderOptions` and `UseNextSanityBlurUpImageBuilderOptions` types.